docs/config-service.rst

```diff
@@ -70,22 +76,16 @@ All configuration options above in the general ``[gcp]`` may be used here. Addit
 ``[gcp.enricher]``
 ~~~~~~~~~~~~~~~~~~
 
-All configuration options above in the general ``[gcp]`` may be used here. Additional Google Compute Engine configuration options are:
-
-
-.. option:: dns_zone="STR"
-
-    `Required`: DNS zone to validate the correctness of A records before publishing. Must be a fully-qualified domain name (FQDN), ending in ``.``, e.g. ``example.com.``.
+All configuration options above in the general ``[gcp]`` may be used here.  If
+``dns_zone`` isn't present here, it must be in ``[gcp]``.
 
 
 ``[gcp.publisher]``
 ~~~~~~~~~~~~~~~~~~~
 
-All configuration options above in the general ``[gcp]`` may be used here. Additional Google Cloud DNS configuration options are:
-
-.. describe:: dns_zone="STR"
-
-    `Required`: DNS zone to validate the correctness of A records before publishing. Must be a fully-qualified domain name (FQDN), ending in ``.``, e.g. ``example.com.``.
+All configuration options above in the general ``[gcp]`` may be used here.  If
+``dns_zone`` isn't present here, it must be in ``[gcp]``.Additional Google
```

Comment:
missing a space before Additional

Reply:
I don't know why this was already hidden, but I fixed it.